mkdocs/docs/experiments/categories/load/k6-loadgen.md

```diff
@@ -0,0 +1,184 @@
+## Introduction
+
+[k6](https://k6.io/) loadgen fault simulates load generation on the target hosts for a specific chaos duration. This fault:
+- Slows down or makes the target host unavailable due to heavy load.
+- Checks the performance of the application or process running on the instance.
+Support [various types](https://grafana.com/docs/k6/latest/testing-guides/test-types/) of load testing (ex. spike, smoke, stress)
+
+!!! tip "Scenario: Load generating with k6"    
+    ![k6-loadgen](../../images/k6-loadgen.png)
+
+## Uses
+
+??? info "View the uses of the experiment" 
+    coming soon
```

Comment:
@namkyu1999 if you have any utility in mind (ones that triggered the creation of this experiment, please feel free to list them here. We are not too proud about the coming soon section on other experiment docs and would like to address that shortly :( )

Reply:
I deleted this "Uses" section temporarily, I will add this section after I write a blog post about this chaos fault on the Litmus blog.